web/main/index.css

```diff
@@ -47,4 +47,8 @@ ol[role='list'] {
   box-sizing: border-box;
   width: 100%;
   padding-inline-start: var(--navbar-width);
+
+  @media (width <=1024px) {
```

Comment:
```suggestion
  @media (width <= 1024px) {
```

Reply:
It seems there is a rule somewhere which prevents me from adding this space in .css files. I haven't figured out where it is yet. I fixed this in Svelte files.

Reply:
Does it remove space on saving the file or during making the commit?

Reply:
It removes space when I try to save the file.

Reply:
It means that it is something in IDE.

My suggestion:
1. Disable Prettier restart IDE and try again.
2. Try to look plugins for something relevant and try to disable it (and restart to check).
3. Disable all plugins.

Reply:
I've corrected it manually